vaas/vaas/vcl/renderer.py

```diff
@@ -183,9 +183,18 @@ def prepare_redirects(self) -> Dict[str, List[Redirect]]:
                 if entries := redirects.get(domain, []):
                     entries.append(redirect)
                 else:
+                    redirect.destination = self.get_redirect_destination(redirect, self.varnish.cluster)
```

Comment:
Im not sure if this is good idea to override field in DB model, if somebody call `.save()` on it, model will update with mapped value. Maybe we should create additional [model method](https://docs.djangoproject.com/en/4.1/topics/db/models/#model-methods) in `Redirect`, where we can implement logic from `get_redirect_destination` ?

Reply:
Fixed, I wrapped Redirect in VclRedirect